Temporary workaround until this GCC bug is fixed:

  -mlong-double-64 should override a previous -mabi=ibmlongdouble
  <https://gcc.gnu.org/bugzilla/show_bug.cgi?id=104208>

diff --git a/sysdeps/ieee754/ldbl-opt/Makefile b/sysdeps/ieee754/ldbl-opt/Makefile
index 1d01846476..0e06abf9f1 100644
--- a/sysdeps/ieee754/ldbl-opt/Makefile
+++ b/sysdeps/ieee754/ldbl-opt/Makefile
@@ -186,8 +186,8 @@ CFLAGS-nldbl-y0.c = -fno-builtin-y0l
 CFLAGS-nldbl-y1.c = -fno-builtin-y1l
 CFLAGS-nldbl-yn.c = -fno-builtin-ynl
 
-tests += test-narrow-macros-ldbl-64 test-nldbl-redirect \
-	 test-redirection-ldbl-64
+#tests += test-narrow-macros-ldbl-64 test-nldbl-redirect \
+#	 test-redirection-ldbl-64
 CFLAGS-test-narrow-macros-ldbl-64.c += -mlong-double-64
 CFLAGS-test-nldbl-redirect.c += -mlong-double-64
 CFLAGS-test-redirection-ldbl-64.c += -mlong-double-64
@@ -198,7 +198,7 @@ endif
 
 # Tests for argp.h functions (reusing the relevant tests from argp/)
 ifeq ($(subdir),argp)
-tests-internal += tst-nldbl-argp
+#tests-internal += tst-nldbl-argp
 
 $(objpfx)tst-nldbl-argp.c: tst-ldbl-argp.c
 	cp $< $@
@@ -209,8 +209,8 @@ endif
 # Tests for err.h and error.h functions (reusing the relevant tests from
 # misc/).
 ifeq ($(subdir), misc)
-tests-internal += tst-nldbl-warn
-tests-internal += tst-nldbl-error
+#tests-internal += tst-nldbl-warn
+#tests-internal += tst-nldbl-error
 
 $(objpfx)tst-nldbl-warn.c: tst-ldbl-warn.c
 	cp $< $@
